nats-server/configuration/clustering/README.md

```diff
@@ -27,10 +32,12 @@ Here is a simple cluster running on the same machine:
 # Check the output of the server for the selected client and route ports.
 ```
 
-The _seed server_ simply declares its client and clustering port. All other servers delegate to the nats-server to auto-select a port that is not in use for both clients and cluster connections, and route to the seed server. Because the clustering protocol gossips members of the cluster, all servers are able to discover other server servers in the cluster. When a server is discovered, the discovering server will automatically attempt to connect to it in order to form a _full mesh_. Typically only one instance of the server will run per machine, so you can reuse the client port \(4222\) and the cluster port \(5222\), and simply the route to the host/port of the seed server.
+The _seed server_ has a client and cluster port specified. Through the use of `-1` as port, the other server are configured to auto-select unused ports for both clients and cluster connections. They then establish a route to the seed server. Because the clustering protocol gossips members of the cluster, all servers are able to discover other server in the cluster. When a server is discovered, the discovering server will automatically attempt to connect to it in order to form a _full mesh_. Typically only one instance of the server will run per machine, so you can reuse the client port \(4222\) and the cluster port \(5222\), and simply the route to the host/port of the seed server.
```

Comment:
> Through the use of `-1` as port

This is not required and to be honest, except for tests where we want to minimize port conflicts if test packages were to be run in parallel (which we no longer do), I would not configure production servers with "random" ports. That may be just me though :-).

Reply:
Agreed, I removed -1 and it's explanation.
Fixed the example that goes with this.
Called out routes option when using multiple seed server